Frontend/appsettings.json

```diff
@@ -28,5 +28,8 @@
   },
   "CypressTestSecret": "secret",
   "AuthenticationExpirationInMinutes": 60,
-  "BetaFeedbackLink": "https://forms.office.com.mcas.ms/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-d2CvT4DIP1JvSu1_jhCZ1xURE4zWktYMjNBM1RNWFcxRElCSVJYMTREUS4u"
+   "BetaFeedbackLink": "https://forms.office.com.mcas.ms/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-d2CvT4DIP1JvSu1_jhCZ1xURE4zWktYMjNBM1RNWFcxRElCSVJYMTREUS4u",
+   "ServiceLink": {
```

Comment:
I think this is here just to let people know that a secret is required for it. Might be worth leaving in, but I don't feel strongly about it.

Reply:
I've switched from `Configuration.GetSection` to `Configuration.GetRequiredSection` so that config problems will throw ASAP, making root causes easier to identify, so had to put it in the config here. (Startup line 56)